io.edcd.EDMarketConnector.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=E:D Market Connector
+Exec=/usr/bin/python3 /app/edmarketconnector/EDMarketConnector.py
```

Comment:
this should be the name of a single executable, found in the path.

Reply:
Changing this to:
```
Exec=/app/edmarketconnector/EDMarketConnector.py
```
*should* work, as the first line of that file is:
```bash
#!/usr/bin/env python3
```

Reply:
On the other hand, reviewing other comments, and checking what's currently in this repo's .desktop file:

1. This has `edmarketconnector.sh`, which is copied to `${FLATPAK_DEST}/bin/edmarketconnector` during installation.
2. That file does the necessary `python3 /app/edmarketconnector/EDMarketConnector.py` (which is flatpak specific).

So I propose to take the .desktop file from this repo and add it into the upstream, but with `Exec=edmarketconnector`.  Documentation .desktop files does suggest that you don't need to fully qualify the value.  We can worry about how an upstream Linux user then gets `edmarketconnector` to be present and do the right thing, but it should mean that this flatpak will both work *and* can use the upstream .desktop file.

Reply:
This will likely be addressed once https://github.com/EDCD/EDMarketConnector/pull/1800 is merged upstream.